### ISSUE TITLE
Doctest traceback formatting error

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#304]
This ReST code:

```
    >>> raise Exception("Something went wrong!")
    Traceback (most recent call last):
    ...
    Exception: Something went wrong!
```

Is both in HTML and Latex outputs rendered with the TraceBack line missing. The reason is the ending colon. However, removing it makes the tests fail. I suspect that this is still handled partly as markup, even though it's a doctest. Putting .. doctest:: before it makes no difference, and neither does separating the test and the output into .. doctest:: and ..testoutput:: blocks.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/304
- Originally reported by: Lennart Regebro
- Originally created at: 2009-12-29T19:13:57.079

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-01-02 14:07:46+00:00_

The reason for the misrendering is that the "..." ellipses are not indented, which triggers a bug in Pygments.

Since this is not Sphinx-related, I'm closing the bug here, but I'll fix the Pygments bug ASAP.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-01-02 14:19:14+00:00_

Okay, it is fixed in Pygments 1.2.1, which is uploading to PyPI right now.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:34+00:00_

Removing version: 0.6.3 (automated comment)
